docs/docs/integrations/vectorstores/astradb.ipynb

```diff
@@ -49,16 +49,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%pip install --upgrade --quiet \"astrapy>=0.7.1\""
+    "pip install --upgrade langchain-astradb"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "2453d83a-bc8f-41e1-a692-befe4dd90156",
    "metadata": {},
    "source": [
-    "_Note: depending on your LangChain setup, you may need to install/upgrade other dependencies needed for this demo_\n",
-    "_(specifically, recent versions of `datasets`, `langchain-openai` and `pypdf` are required, along with `langchain-community`)._"
+    "_**Note.** these are the packages required to run the full demo on this page. Depending on your LangChain setup, you may not have all of them already:_ `pip install langchain langchain-openai datasets pypdf`."
```

Comment:
We could have these in a `pip install` cell.
It won't harm to run it if they're already installed.

Reply:
Good idea, done! Thanks